cmd/mdatagen/loader.go

```diff
@@ -79,6 +79,11 @@ func (mvt *ValueType) UnmarshalText(text []byte) error {
 
 // String returns capitalized name of the ValueType.
 func (mvt ValueType) String() string {
+	// Temporary assume that empty value type is string because "type" is an optional field for data point attributes.
+	// TODO: Make the "type" attribute a required field for data point attributes as it's done for resource attributes.
```

Comment:
Noting that this is solved when https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16420 is merged.

Reply:
Resolved